src/features/Instructor/Availability/index.jsx

```diff
@@ -47,32 +48,13 @@ const Availability = () => {
     });
   }, [setRangeDates]);
 
-  const handleDeleteEvent = async (event) => {
-    try {
-      const params = event?.recurrence ? {
-        delete_occurrence: true,
-        start_occurrence: event?.start,
-      } : {};
-
-      await deleteEvent(event.id, params);
-    } catch (error) {
-      logError(error);
-    } finally {
-      dispatch(fetchEventsData(rangeDates));
-    }
-  };
-
-  const handleDeleteMultipleEvents = async (event) => {
-    try {
-      await deleteEvent(event.id);
-    } catch (error) {
-      logError(error);
-    } finally {
-      dispatch(fetchEventsData(rangeDates));
-    }
-  };
-
-  const handleEvent = async (eventData, isEdit = false) => {
+  /**
+   * Edit multiple recurrences or create a new event according the situation.
```

Comment:
```suggestion
   * Edit multiple recurrences or create a new event, depending on the situation.
```

"depending on the situation" is the more appropriate choice because it suggests a conditional relationship between the action (editing or creating) and the specific circumstance.